lib/nanoc/cli/commands/create-site.rb

```diff
@@ -281,7 +282,7 @@ def run
 
       # Check whether site exists
       if File.exist?(path)
```

Comment:
Can you move the `unless options[:force]` next to the condition here?

```ruby
if File.exist?(path) && !options[:force]
```

The `options[:force]` will otherwise be overlooked too easily, I feel.